recipes/python/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: python
+  version: 3.5.1
+
+source:
+  fn: Python-3.5.1.tgz
+  url: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tgz
+  md5: be78e48cdfc1a7ad90efff146dce6cfe
+  patches:
+    - condaforge_version.patch
+
+build:
+  no_link: 
+    - bin/python
+  number: 0
+
+requirements:
+  build:
+    - bzip2 1.0.*       # [unix]
```

Comment:
Please feel free to update https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies with these new pinned dependencies.